config/default.yml

```diff
@@ -333,6 +333,17 @@ Rails/MailerName:
   Include:
     - app/mailers/**/*.rb
 
+Rails/MatchRoute:
+  Description: >-
+                  Don’t use match to define any routes unless there is need to map multiple request types
```

Comment:
I think "there is **a** need" would read slightly better.

Also "using **the** :via option"

Reply:
Thanks! 
I'm not a native speaker, so I'm expecting a lot of mistakes. I have to study English better 😄